scripts/install-containerd

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# install and start containerd daemon
+
+set -eu -o pipefail
+
+ARCH="$(go env GOARCH)"
+OS="$(go env GOOS)"
+CONTAINERD_VERSION="$(go list -f {{.Module.Version}} -find github.com/containerd/containerd)"
+
+# check if already installed and start
+ctrd=$(which containerd 2> /dev/null)
+if [ "$ctrd" != "" ]; then
+    echo "containerd installed to $ctrd"
+    $ctrd &
+    exit 0
+fi
```

Comment:
Ubuntu GHA runner image is having containerd v1.6.22 pre-installed. I'm thinking we should update go.mod and download the containerd version specified there.